get-started/codelab/index.md

```diff
@@ -0,0 +1,1301 @@
+          new IconButton(icon: new Icon(Icons.list), onPressed: _pushSaved),
+        ],
+      ),
+      body: _buildSuggestions(),
+    );
+  }
+
+  Widget _buildSuggestions() {
+    return new ListView.builder(
+      padding: const EdgeInsets.all(16.0),
+      itemBuilder: (context, i) {
+        if (i.isOdd) return new Divider();
+
+        final index = i ~/ 2;
+        if (index >= _suggestions.length) {
+          _suggestions.addAll(generateWordPairs().take(10));
+        }
+        return _buildRow(_suggestions[index]);
+      },
+    );
+  }
+
+  Widget _buildRow(WordPair pair) {
+    final alreadySaved = _saved.contains(pair);
+    return new ListTile(
+      title: new Text(
+        pair.asPascalCase,
+        style: _biggerFont,
+      ),
+      trailing: new Icon(
+        alreadySaved ? Icons.favorite : Icons.favorite_border,
+        color: alreadySaved ? Colors.red : null,
+      ),
+      onTap: () {
+        setState(
+          () {
+            if (alreadySaved) {
+              _saved.remove(pair);
+            } else {
+              _saved.add(pair);
+            }
+          },
+        );
+      },
+    );
+  }
+
+  void _pushSaved() {
+    Navigator.of(context).push(
+      new MaterialPageRoute(
+        builder: (context) {
+          final tiles = _saved.map(
+                (pair) {
+              return new ListTile(
+                title: new Text(
+                  pair.asPascalCase,
+                  style: _biggerFont,
+                ),
+              );
+            },
+          );
+          final divided = ListTile
+              .divideTiles(
+            context: context,
+            tiles: tiles,
+          )
+              .toList();
+
+          return new Scaffold(
+            appBar: new AppBar(
+              title: new Text('Saved suggestions'),
+            ),
+            body: new ListView(children: divided),
+          );
+        },
+      ),
+    );
+  }
+}
+{% endprettify %}
+
+---
+
+# Well done!
+
+You've written an interactive Flutter app that runs on both iOS and Android.
+In this codelab, you've:
+
+* Created a Flutter app from the ground up.
+* Written Dart code.
+* Leveraged an external, third party library.
+* Used hot reload for a faster development cycle.
+* Implemented a stateful widget, adding interactivity to your app.
+* Created a lazily loaded, infinite scrolling list displayed with a
+  ListView and ListTiles.
+* Created a route and added logic for moving between the home route
+  and the new route.
+* Learned about Themes.
+
+Here are some resources you might find useful:
+
+{% comment %}
+Once the "Flutter for React" doc (etc) are done, add here.
+{% endcomment %}
+
+* [Flutter API docs](https://docs.flutter.io/)
+* [Building Layouts in Flutter](/tutorials/layout/) tutorial
+* [Add Interactivity](/tutorials/interactive/) tutorial
+* [Flutter Cookbook](/cookbook/)
+* [From Java to Dart](https://codelabs.developers.google.com/codelabs/from-java-to-dart/#0) codelab
```

Comment:
Is the gitter channel (https://gitter.im/flutter/flutter) considered an official source for help?

Reply:
I believe so, but Filip doesn't want me adding  community links. They will be added to the footer soon.